lit_tests/kernel/wave/attention/extend_attention.py

```diff
@@ -314,10 +314,11 @@ def test_causal_extend_attention_32x32x8():
 
     # CHECK-LABEL:       test_causal_extend_attention_32x32x8
     # CHECK:             func.func @extend_attention
+    # CHECK-DAG:            %[[C4608:.*]] = arith.constant 4608 : index
     # CHECK-DAG:            stream.binding.subspan %{{.*}}[%{{.*}}] : !stream.binding -> memref<?x16x64xf16, strided<[1024, 64, 1], offset: ?>>
-    # CHECK-DAG:            %[[ALLOC0:.*]] = memref.alloc() : memref<8704xi8, #gpu.address_space<workgroup>>
-    # CHECK-DAG:            %[[ALLOC1:.*]] = memref.view %[[ALLOC0]][%[[C0]]][] : memref<8704xi8, #gpu.address_space<workgroup>> to memref<32x1x68xf16, #gpu.address_space<workgroup>>
-    # CHECK-DAG:            %[[ALLOC2:.*]] = memref.view %[[ALLOC0]][%[[C4352]]][] : memref<8704xi8, #gpu.address_space<workgroup>> to memref<1x32x68xf16, #gpu.address_space<workgroup>>
+    # CHECK-DAG:            %[[ALLOC0:.*]] = memref.alloc() : memref<8960xi8, #gpu.address_space<workgroup>>
```

Comment:
why does the shape change?

Reply:
shared alloc got untransposed and combining with the padding resulted in different overall number of elements

Reply:
makes sense